src/assets/scss/_articleCard.scss

```diff
@@ -18,6 +18,7 @@
   div {
     cursor: pointer;
     padding: 12px;
+    -webkit-column-break-inside: avoid;
```

Comment:
using `-webkit-*` means you are targeting a chrome browser. Have you tried testing on other browsers to see if the column break inside is avoided? besides what does this attribute do?

Reply:
This was a suggestion from our P.O. One article card was breaking into pieces which wasn't good.

Reply:
It avoid the break inside the article card at the home page. I tried it with Safari and everything is working well.